### PR TITLE
Бочкарев - Лабораторная работа #1

### PR DESCRIPTION
**Pull request by TibidoX**
Review @danielbulgakov @imterinaa

### CODE REVIEW

**Review by danielbulgakov — APPROVED** [review]
🐐

**Review by imterinaa — APPROVED** [review]
GREAT=)

File: modules/complex-number/test/test_BOCHKAREV_VLADIMIR_complex_number.cpp [hunk 1]
```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 Bochkarev Vladimir
+
+#include <gtest/gtest.h>
+
+#include "include/complex_number.h"
+
+TEST(Bochkarev_Vladimir_ComplexNumberTest, Test_Selfsubstruct) {
+    // Arrange
+    ComplexNumber a(1, 1);
+    ComplexNumber ans(0.0, 0.0);
```
Comment (dkurt) [inline modules/complex-number/test/test_BOCHKAREV_VLADIMIR_complex_number.cpp:10]: Should be non-zero

Comment (TibidoX) [reply modules/complex-number/test/test_BOCHKAREV_VLADIMIR_complex_number.cpp:10]: @dkurt  how can you subtract a number from the exact same number without getting 0?

Comment (dkurt) [reply modules/complex-number/test/test_BOCHKAREV_VLADIMIR_complex_number.cpp:10]: Why we even should subtract number from itself?

I mean in some cases not initialized class fields might be zeros, so we can miss it by comparing with all-zero number.

File: modules/complex-number/test/test_BOCHKAREV_VLADIMIR_complex_number.cpp [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    // Arrange
+    ComplexNumber a(5, 5);
+    ComplexNumber b(5, -5);
+    ComplexNumber ans(0, 1);
+
+    // Act
+    ComplexNumber result = a / b;
+
+    // Assert
+    ASSERT_EQ(ans.getIm(), result.getIm());
```
Comment (RomanMitin) [inline modules/complex-number/test/test_BOCHKAREV_VLADIMIR_complex_number.cpp:59]: Use ASSERT_DOUBLE_EQ, instead  ASSERT_EQ.

File: modules/complex-number/test/test_BOCHKAREV_VLADIMIR_complex_number.cpp (hunk 1 above)
Comment (dkurt) [reply modules/complex-number/test/test_BOCHKAREV_VLADIMIR_complex_number.cpp:10]: Friendly reminder

Comment (TibidoX) [reply modules/complex-number/test/test_BOCHKAREV_VLADIMIR_complex_number.cpp]: @dkurt I'm ready

**Pull request merged**